session/list_test.go

```diff
@@ -0,0 +1,104 @@
+package session
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/joshmedeski/sesh/config"
+)
+
+type mockConfigDirectoryFetcher struct {
+	dir string
+}
+
+func (m *mockConfigDirectoryFetcher) GetUserConfigDir() (string, error) {
+	return m.dir, nil
+}
+
+type MockSeshConfig struct {
+	Name     string
+	Contents string
+	Imports  string
+}
+
+func prepareSeshConfig(t *testing.T, mockSeshConfigs []MockSeshConfig) string {
```

Comment:
this is a straight up copy from the function in config_test.go

Maybe it might be useful to put it into a testUtils package?